README.md

```diff
@@ -3,7 +3,19 @@
 
 A music player for the busy ones. [http://headsetapp.co](http://headsetapp.co)
 
-## Building for other systems
+## Install on debian based OS:
+```
+$ wget -q https://packagecloud.io/headsetapp/headset-electron/gpgkey -O- | sudo apt-key add -
+$ echo "deb https://packagecloud.io/headsetapp/headset-electron/ubuntu/ trusty main" | sudo tee /etc/apt/sources.list.d/headsetapp_headset-electron.list
+```
+
+After those two commands are run, just install headset with:
+
+```
+$ sudo apt install headset
```

Comment:
just in case people don't update their repositories, which is common practice (i.e. new people to linux), we should change the line to:
`sudo apt update && sudo apt install headset`